### PR TITLE
Removed Wrapper.dart and database.dart

### PR DESCRIPTION
**Pull request by atharv-bhadange**
Also removed their imports in other files

### CODE REVIEW

**Review by Saransh-cpp — APPROVED** [review]
Thanks, @atharv-bhadange! Could you resolve the conflicts and let me know which program are you a part of? Let me know if you have any difficulties.

Suggestions -

- Never commit on the master/main branch, always create a new branch and make a PR from that branch.

**Comment by atharv-bhadange** [comment]
I have resolved the conflicts. I am contributing to this project as a part of the SWoC 2.0 program. 
I'll keep in mind to commit from a different branch.

File: lib/Screens/Home.dart [hunk 1]
```diff
@@ -39,377 +35,328 @@ class _HomeState extends State<Home> {
     final userProvider = Provider.of<UserProvider>(context);
     final productProvider = Provider.of<ProductProvider>(context);
 
-    return SingleChildScrollView(
-      child: SafeArea(
+    return SafeArea(
```
Comment (Saransh-cpp) [inline lib/Screens/Home.dart:38]: @atharv-bhadange I think you made a mistake while resolving conflicts, we want the `SingleChildScrollView` to be there.

Comment (atharv-bhadange) [reply lib/Screens/Home.dart:38]: The `SingleChildScrollView` is giving some error. I'm not able to resolve it, could you help me with that?

Comment (Saransh-cpp) [reply lib/Screens/Home.dart:38]: Yes, post the errors below or you can push the code and I'll take a look.

Comment (atharv-bhadange) [reply lib/Screens/Home.dart:38]: image

This is the error I am getting and the contents on the home screen are not visible.

**Review by Saransh-cpp — APPROVED** [review]
Thanks, @atharv-bhadange! The issue pointed out by you was created here - #39. This PR now solves 2 issues, I'll mark you accordingly.

Also, the other changes are just formatting changes, right?

**Comment by atharv-bhadange** [comment]
Yes, the other changes are just formatting changes. Thanks for your help.

**Comment by Saransh-cpp** [comment]
@all-contributors please add @atharv-bhadange for code

**Pull request merged**